common/views/components/Checkbox/Checkbox.js

```diff
@@ -71,6 +71,10 @@ type CheckboxWithoutLabelProps = {|
 type CheckboxProps = {|
   ...CheckboxWithoutLabelProps,
   text: string,
+  checked: boolean,
+  name: string,
+  onChange: ?() => void,
```

Comment:
If we send a value, doesn't react need you to send an `onChange`?

Reply:
We do. Good point. Will update.